esp-println/src/logger.rs

```diff
@@ -1,49 +1,42 @@
-use core::str::FromStr;
-
-use log::LevelFilter;
-
 use super::println;
 
-const LOG_TARGETS: Option<&'static str> = option_env!("ESP_LOGTARGETS");
+#[cfg(not(host_is_windows))]
+include!(concat!(env!("OUT_DIR"), "/log_filter.rs"));
```

Comment:
Is it possible to also concat https://doc.rust-lang.org/std/path/constant.MAIN_SEPARATOR_STR.html instead of the cfg here? I'm not sure how macros are expanded, but I'm guessing this might fail to compile.

Reply:
That unfortunately isn't possible (and it's a shame we need to do this at all)